presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -606,6 +606,33 @@ private void dropTableIfExists(String tableName)
         computeExpected("DROP TABLE IF EXISTS " + tableName, ImmutableList.of(BIGINT));
     }
 
+    @Test
+    public void testInsertIntoUnpartitionedTable()
+    {
+        Session session = Session.builder(getSession())
+                .setSystemProperty("table_writer_merge_operator_enabled", "false")
+                .setCatalogSessionProperty("hive", "collect_column_statistics_on_write", "false")
+                .build();
+
+        // Generate temporary table name.
```

Comment:
This code repeats. Consider extracting a helper method.

Reply:
+1